src/jit/compiler.h

```diff
@@ -8186,13 +8186,15 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
         bool compPublishStubParam : 1; // EAX captured in prolog will be available through an instrinsic
         bool compRetBuffDefStack : 1;  // The ret buff argument definitely points into the stack.
 
-        var_types compRetType;       // Return type of the method as declared in IL
-        var_types compRetNativeType; // Normalized return type as per target arch ABI
-        unsigned  compILargsCount;   // Number of arguments (incl. implicit but not hidden)
-        unsigned  compArgsCount;     // Number of arguments (incl. implicit and     hidden)
-        unsigned  compRetBuffArg;    // position of hidden return param var (0, 1) (BAD_VAR_NUM means not present);
-        int compTypeCtxtArg; // position of hidden param for type context for generic code (CORINFO_CALLCONV_PARAMTYPE)
-        unsigned       compThisArg; // position of implicit this pointer param (not to be confused with lvaArg0Var)
+        var_types compRetType;          // Return type of the method as declared in IL
+        var_types compRetNativeType;    // Normalized return type as per target arch ABI
+        unsigned  compILargsCount;      // Number of arguments (incl. implicit but not hidden)
+        unsigned  compArgsCount;        // Number of arguments (incl. implicit and     hidden)
+        unsigned  compArgRegCount;      // Number of integer locals
+        unsigned  compOtherArgRegCount; // Number of multireg args
```

Comment:
These should be ifdef'd under `FEATURE_MULTIREG_ARGS`. Also, is it the number of integer locals or the number of integer arguments in registers?